Development

AjithBillava opened the pull request.

AjithBillava committed:
feat: added responsiveness

AjithBillava committed:
updated responsiveness

AjithBillava committed:
Update documentation.html

AjithBillava committed:
updated mobile-nav height

AjithBillava committed:
updated landing page responsiveness

AjithBillava committed:
feat: added code snippets

AjithBillava committed:
update: fixed float btn

aneenajohn reviewed (changes requested):
Amazing work @AjithBillava ! Overall, Component Library looks awesome!
Here are the few suggestions to improve the component library.

1)Include get started page , so that users will get to know how to use your library.
2)Add a close icon to the modal component.
3)Change the cursor to pointer on the toast close icon.
4)Add a shade litter darker on the buttons for hover state to make the hover state more prominent.

AjithBillava committed:
updated acc to Aneena's review

1)Include get started page , so that users will get to know how to use your library.
2)Add a close icon to the modal component.
3)Change the cursor to pointer on the toast close icon.
4)Add a shade litter darker on the buttons for hover state to make the hover state more prominent.

AjithBillava commented:
> Amazing work @AjithBillava ! Overall, Component Library looks awesome!
> Here are the few suggestions to improve the component library.
> 
> 1)Include get started page , so that users will get to know how to use your library.
> 2)Add a close icon to the modal component.
> 3)Change the cursor to pointer on the toast close icon.
> 4)Add a shade litter darker on the buttons for hover state to make the hover state more prominent.

Thanks for the comment, I have incorporated the changes
